ts/Extensions/StaticScale.ts

```diff
@@ -98,10 +98,12 @@ Chart.prototype.adjustHeight = function (): void {
 
                 diff = height - chart.plotHeight;
 
-                if (Math.abs(diff) >= 1) {
-                    chart.plotHeight = height;
-                    chart.redrawTrigger = 'adjustHeight';
-                    chart.setSize(void 0, chart.chartHeight + diff, animate);
+                if (!chart.scrollablePixelsX && !chart.scrollablePixelsY) {
```

Comment:
Do we need to check both the X and the Y direction? Isn't it enough to block static scale only in the vertical direction?

Reply:
You are right! There is no need to check both.